src/fields/multi_field.impl.hh

```diff
@@ -149,14 +149,24 @@ void MultiField<spacedim, Value>::set_up_components() {
     		sub_fields_[i_comp].name_ = this->shared_->comp_names_[i_comp] + "_" + name();
     		sub_fields_[i_comp].shared_->input_name_ = name();
     	}
-
-    	sub_fields_[i_comp].set_input_list(this->shared_->input_list_);
     }
 }
 
 
 
 template<int spacedim, class Value>
+void MultiField<spacedim,Value>::set_input_list(const Input::Array &list) {
```

Comment:
I think that this will not work. Do we have some unit tests for multifields?
Anyway we can postpone it when we will deal with necessary changes in the MultiFieldFactory.